src/mongocxx/options/find.hpp

```diff
@@ -128,6 +127,29 @@ class MONGOCXX_API find {
     const stdx::optional<std::int32_t>& limit() const;
 
     ///
+    /// The maximum amount of time for the server to wait on new documents to satisfy a tailable
+    /// cursor query. This only applies to a TAILABLE_AWAIT cursor. When the cursor is not a
+    /// TAILABLE_AWAIT cursor, this option is ignored. The default on the server is to wait for one
+    /// second.
+    ///
+    /// @note On servers < 3.2, this option is ignored.
+    ///
+    /// @param max_await_time_ms
+    ///   The max amount of time (in milliseconds) to wait for new documents.
+    ///
+    /// @see http://docs.mongodb.org/manual/reference/operator/meta/maxTimeMS
+    ///
+    void max_await_time_ms(std::int64_t max_await_time_ms);
```

Comment:
Should the argument here be a std::chrono duration? It seems like it would be much more idiomatic. Do we have other places where we are setting timeouts?